docs/source/examples/mg-mode-part-iii.ipynb

```diff
@@ -0,0 +1,1502 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f35ed7ce160>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "universes['Root'].plot(center=(length / 2., length / 2., 0.),\n",
+    "                       pixels=(500, 500), width=(length, length),\n",
+    "                       color_by='material',\n",
+    "                       colors={materials['Fuel']: (1., 0., 0.),\n",
+    "                               materials['Gad']: (1., 1., 0.),\n",
+    "                               materials['Zirc2']: (0.5, 0.5, 0.5),\n",
+    "                               materials['Water']: (0.0, 0.0, 1.0),\n",
+    "                               materials['B4C']: (0.0, 0.0, 0.0),\n",
+    "                               materials['Steel']: (0.4, 0.4, 0.4)})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Looks pretty good to us!\n",
+    "\n",
+    "We now must create a geometry that is assigned a root universe and export it to XML."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create Geometry and set root Universe\n",
+    "geometry = openmc.Geometry()\n",
+    "geometry.root_universe = universes['Root']\n",
+    "# Export to \"geometry.xml\"\n",
+    "geometry.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the geometry and materials finished, we now just need to define simulation parameters, including how to run the model and what we want to learn from the model (i.e., define the tallies). We will start with our simulation parameters in the next block.\n",
+    "\n",
+    "This will include setting the run strategy, telling OpenMC not to bother creating a `tallies.out` file, and limiting the verbosity of our output to just the header and results to not clog up our notebook with results from each batch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# OpenMC simulation parameters\n",
+    "batches = 2000\n",
+    "inactive = 200\n",
```

Comment:
Ditto, I doubt it takes more than about 20 batches to converge the Shannon entropy for this problem, though you could check that to convince yourself of this if you wish. Reduce the inactive batches to 20 or 30 to shave off a little runtime for this notebook.